.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/elixir:1.7
+      - image: circleci/elixir:1.10
```

Comment:
For a generally useful library like this, I prefer to keep the Elixir version to as an early a version as possible. Doing so in the CI ensures that we retain support for as wide a set of Elixir versions as possible